.cirrus.yml

```diff
@@ -285,6 +346,49 @@ task:
   test_all_script:
     - ./dev/bots/deploy_gallery.sh
 
+task:
+  use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
+  osx_instance:
+    image: mojave-xcode-10.1
+  depends_on:
+    - analyze
+  env:
+    CIRRUS_WORKING_DIR: "/tmp/flutter sdk"
+    COCOAPODS_DISABLE_STATS: true
```

Comment:
Maybe it's more readable to name it "COCOAPODS_DISABLED"?

Reply:
That's an environment variable specifically for Cocoapods, we can't change the name of it.  It disables metrics reporting, which is slow and not needed for CI anyway.